docs/userguide/networking/overlay-security-model.md

```diff
@@ -0,0 +1,86 @@
+<!--[metadata]>
++++
+title = "Docker 1.12 overlay network security model"
+description = "Docker 1.12 Overlay Network Security Model"
+keywords = ["network, docker, documentation, user guide, multihost, swarm mode", "overlay"]
+[menu.main]
+parent = "smn_networking"
+weight=-2
++++
+<![end-metadata]-->
+
+## Note on Docker 1.12 overlay network security model
```

Comment:
Looks like this section is about more than just "overlay networks" in swarm mode (also worker/manager relation and security)